src/selectors/layout.js

```diff
@@ -38,26 +39,34 @@ export const getSidebarWidth = (visibleSidebar, outerChartWidth) => {
   return sidebarWidth.closed;
 };
 
+/**
+ * Return the displayed width of the meta sidebar
+ */
+export const getMetaSidebarWidth = visibleMetaSidebar =>
+  visibleMetaSidebar ? metaSidebarWidth.open : metaSidebarWidth.closed;
+
 /**
  * Convert the DOMRect into an Object, mutate some of the properties,
  * and add some useful new ones
  */
 export const getChartSize = createSelector(
-  [getVisibleSidebar, state => state.chartSize],
-  (visibleSidebar, chartSize) => {
+  [getVisibleSidebar, getVisibleMetaSidebar, state => state.chartSize],
+  (visibleSidebar, visibleMetaSidebar, chartSize) => {
     const { left, top, width, height } = chartSize;
     if (!width || !height) {
       return {};
     }
-    const sidebarWidth = getSidebarWidth(visibleSidebar, width);
+    const currentSidebarWidth = getSidebarWidth(visibleSidebar, width);
+    const currentMetaWidth = getMetaSidebarWidth(visibleMetaSidebar);
     return {
       left,
       top,
       outerWidth: width,
       outerHeight: height,
-      width: width - sidebarWidth,
+      width: width - currentSidebarWidth - currentMetaWidth,
       height,
-      sidebarWidth
+      sidebarWidth: currentSidebarWidth,
+      metaSidebarWidth: currentMetaWidth
```

Comment:
I don't understand why you've renamed these to add 'current' to the names, as they just get renamed again in the function output anyway? Can you please explain the distinction?

Reply:
Ah yes this can now just be reverted as it's no longer necessary.